tiatoolbox/utils/misc.py

```diff
@@ -712,6 +712,50 @@ def string_to_tuple(in_str):
     return tuple(substring.strip() for substring in in_str.split(","))
 
 
+def prepare_file_dir_cli(img_input, output_path, file_types, mode, sub_dirname):
+    """Prepares cli for running code on multiple files or a directory.
+
+    Checks for existing directories to run tests.
+    Converts file path to list of file paths or
+    creates list of file paths if input is a directory.
+
+    Args:
+        img_input (str or pathlib.Path): file path to images.
+        output_path (str or pathlib.Path): output directory path.
+        file_types (str): file types to process using cli.
+        mode (str): wsi or tile mode.
+        sub_dirname (str): name of sub directory to save output.
+
+    Returns:
+        files_all (list): list of file paths to process.
+        output_path (pathlib.Path): Updated output path.
```

Comment:
Very minor but there is inconsistent capitalisation here.

Reply:
resolved in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/220/commits/625222fdb85414b351af4f908c31cec021d1307b